### DISCUSSION TITLE
New plans: Multiple PR assignees versus reviewers

### DISCUSSION THREAD

**Discussion by lukebaker** [community/community#22132] [New to GitHub]
<p>With the new plans, it appears there’s conflicting or incomplete information with regard to the limitations of private repositories on free plans. For a particular pull request in a private repository, can GitHub Free for organzations assign multiple users to that pull request? Can they add multiple users as reviewers of that pull request?</p>
<p>On the <a href="https://github.com/pricing" rel="nofollow noopener">pricing page</a>, it indicates that free plans may only have “Multiple pull request assignees” for public repositories. While on the <a href="https://help.github.com/en/github/getting-started-with-github/githubs-products" rel="nofollow noopener">products page</a>, it lists “Multiple pull request reviewers” (and no mention of assignees) as a feature of GitHub Team plan in private repositories. As I understand it, “assignees” and “reviewers” refer to different ways to interact with a pull request.</p>
<p>When I test in my GitHub Free for users account, I’m able to add more than one assignee and reviewer to a particular pull request in a private repository. The interface indicates I can add 10 and 15 for those respectively, which appears in conflict with the information on the pricing and product pages.</p>

**Comment by matthewisabel** [comment, marked as answer]
<p>Hey <a class="mention" href="/lukebaker">@lukebaker</a> , sorry about the doc confusion and thanks for raising here. I’m on the product team and I’ll get these two areas fixed up and in sync. On GitHub for Free for organization, you’re limited to one assignee on issue assignees, PR assignees, and PR reviewers.</p>
<p>Sorry about the confusion and let me know if I can help with anything else!</p>

**Comment by pauladler** [comment]
<p>Hi <a class="mention" href="/matthewisabel">@matthewisabel</a>, That’s really disappointing. We just swapped over from the old Pro plan (we had been paying for over 5 years prior to the changes in April 2020) to the new Teams free plan and have found that we can no longer invite multiple reviewers to review a PR. This really should be highlighted strongly in the documentation, because we would not have swapped over to the Teams free plan if we had known this.<br>
Now we are not sure what to do, because it would cost us a lot of money to pay for the Teams plan, but living with this new limitation is going to be cumbersome.</p>
